x/muxer/tests/muxer_test.go

```diff
@@ -68,6 +69,20 @@ func TestSimpleDispatch(t *testing.T) {
 	)
 }
 
+func TestCheckConfigErrorNotDuplicated(t *testing.T) {
+	var m muxer.DispatchTable
+	m.Resources = map[string]int{
+		"test:mod:A": 0,
+		"test:mod:B": 1,
+	}
+	req := `{}`
+	mux(t, m).replay(
+		exchange("/pulumirpc.ResourceProvider/CheckConfig", req, "{}", `["[\"myerr\"]"]`,
+			part(0, req, "{}", `["myerr"]`),
```

Comment:
You have created a variable to represent `"{}"`, but then also included the literal `"{}"`. Can we stick to literals or variables here?

Reply:
Yup, mb, I think `req` evolved while I was editing this and I didn't fold them in.